packages/katello/rubygem-foreman_inventory_upload/rubygem-foreman_inventory_upload.spec

```diff
@@ -35,71 +35,25 @@ Provides: foreman-plugin-%{plugin_name} = %{version}
 # end specfile generated dependencies
 
 # start package.json devDependencies BuildRequires
-BuildRequires: %{?scl_prefix}npm(babel-cli) >= 6.10.1
-BuildRequires: %{?scl_prefix}npm(babel-cli) < 7.0.0
-BuildRequires: %{?scl_prefix}npm(babel-core) >= 6.26.3
-BuildRequires: %{?scl_prefix}npm(babel-core) < 7.0.0
-BuildRequires: %{?scl_prefix}npm(babel-loader) >= 7.1.1
-BuildRequires: %{?scl_prefix}npm(babel-loader) < 8.0.0
-BuildRequires: %{?scl_prefix}npm(babel-plugin-lodash) >= 3.3.4
-BuildRequires: %{?scl_prefix}npm(babel-plugin-lodash) < 4.0.0
-BuildRequires: %{?scl_prefix}npm(babel-plugin-syntax-dynamic-import) >= 6.18.0
-BuildRequires: %{?scl_prefix}npm(babel-plugin-syntax-dynamic-import) < 7.0.0
-BuildRequires: %{?scl_prefix}npm(babel-plugin-transform-class-properties) >= 6.24.1
-BuildRequires: %{?scl_prefix}npm(babel-plugin-transform-class-properties) < 7.0.0
-BuildRequires: %{?scl_prefix}npm(babel-plugin-transform-object-assign) >= 6.8.0
-BuildRequires: %{?scl_prefix}npm(babel-plugin-transform-object-assign) < 7.0.0
-BuildRequires: %{?scl_prefix}npm(babel-plugin-transform-object-rest-spread) >= 6.8.0
-BuildRequires: %{?scl_prefix}npm(babel-plugin-transform-object-rest-spread) < 7.0.0
-BuildRequires: %{?scl_prefix}npm(babel-preset-env) >= 1.7.0
-BuildRequires: %{?scl_prefix}npm(babel-preset-env) < 2.0.0
-BuildRequires: %{?scl_prefix}npm(babel-preset-react) >= 6.5.0
-BuildRequires: %{?scl_prefix}npm(babel-preset-react) < 7.0.0
-BuildRequires: %{?scl_prefix}npm(identity-obj-proxy) >= 3.0.0
-BuildRequires: %{?scl_prefix}npm(identity-obj-proxy) < 4.0.0
+BuildRequires: %{?scl_prefix}npm(@babel/core) >= 7.7.0
+BuildRequires: %{?scl_prefix}npm(@babel/core) < 8.0.0
+BuildRequires: %{?scl_prefix}npm(@theforeman/builder) >= 4.0.2
+BuildRequires: %{?scl_prefix}npm(@theforeman/builder) < 5.0.0
+BuildRequires: %{?scl_prefix}npm(@theforeman/eslint-plugin-foreman) >= 4.0.2
+BuildRequires: %{?scl_prefix}npm(@theforeman/eslint-plugin-foreman) < 5.0.0
+BuildRequires: %{?scl_prefix}npm(@theforeman/stories) >= 4.0.2
+BuildRequires: %{?scl_prefix}npm(@theforeman/stories) < 5.0.0
+BuildRequires: %{?scl_prefix}npm(@theforeman/test) >= 4.0.2
+BuildRequires: %{?scl_prefix}npm(@theforeman/test) < 5.0.0
```

Comment:
AFAIK they should be ignored

Reply:
should we erase those lines? 
or is it a bug in `koji` which should ignore `devDependencies`

Reply:
If these are not needed for building, they should be added to the list in https://github.com/theforeman/foreman-packaging/blob/rpm/develop/update-requirements#L10

Reply:
actually they are listed there,
maybe I am using an old `gem2rpm` version.. I will try again

Reply:
Typically it's a sign you need to rebase on a more recent `rpm/develop`.